src/os/bluestore/BlueStore.cc

```diff
@@ -5778,7 +5719,6 @@ int BlueStore::_do_read(
   }
 
   utime_t start = ceph_clock_now();
-  o->flush();
```

Comment:
That's okay.  The OSD layer above this doesn't read data until it's gotten the onreadable callback.